### PR TITLE
Store mode per board and listen for changes to synced mode

### PR DESCRIPTION
**Pull request by MareinK**
Let me know if it needs improving!

### CODE REVIEW

File: js/init.js [hunk 1]
```diff
@@ -1,32 +1,48 @@
 function syncState() {
-  chrome.storage.sync.get('classList', function(result) {
+  let board_id = location.pathname.split('/')[2];
```
Comment (sudonatalie) [inline js/init.js]: Could you rename board_id to boardId, I think all the other var names here are using camel case.

File: js/init.js [hunk 2]
```diff
@@ -1,32 +1,48 @@
...
+    if (result[storageKey]) {
+      board.classList.add(result[storageKey]);
     }
   });
 }
 
+chrome.storage.onChanged.addListener(function (changes, namespace) {
+  let board = document.getElementById('board');
+  let board_id = location.pathname.split('/')[2];
+  let storageKey = 'classList_' + board_id;
```
Comment (sudonatalie) [inline js/init.js]: Since the storageKey construction is repeated in a few places, can you extract it to a function?

File: js/init.js [hunk 3]
```diff
@@ -1,32 +1,48 @@
...
   const classMixed = 'layout-trello-mixed';
 
   let board = document.getElementById('board');
+  let board_id = location.pathname.split('/')[2];
+  let storageKey = 'classList_' + board_id;
 
   if (board.classList.contains(classMixed)) {
     board.classList.remove(classMixed);
     board.classList.add(classVertical);
     chrome.storage.sync.set({
```
Comment (sudonatalie) [inline js/init.js]: Does this line now trigger the `chrome.storage.onChanged` listener, causing the class names to be added and removed a second time? If so, we can probably remove some unnecessary code in `toggleLayout`, but that's not super important.

Comment (MareinK) [reply js/init.js]: You're right! Done.

**Comment by MareinK** [comment]
Thanks for the tips! I implemented them (except for the migration logic). Let me know if there is anything else.

**Pull request merged**